woodpecker/woodpecker.go

```diff
@@ -121,12 +136,22 @@ func (c *Config) Valid() error {
 		return errors.New("One of FetchConfig or SubmitConfig must not be nil")
 	}
 
+	if c.SubmitConfig == nil && c.InclusionConfig != nil {
```

Comment:
This is slightly confusing to me, it feels like this should just be incorporated into the above check so one of fetch/submit/inclusion must be present, but not specific configuration of the three...

Reply:
I didn't think it made sense to have just an `InclusionConfig` since the certs that are checked for inclusion come from the `SubmitConfig`.

Reply:
Yeah that makes sense, I can imagine you might want to i.e. turn off submissions if there is a giant backlog and just want to check inclusion of the existing stuff (i.e. in a case when submissions are no longer being accepted but for whatever reason sequencing is still happening).

This is probably fine though, I can only think of weird corner cases where you wouldn't want to do it this way.